src/Components/CurrWeatherCard/CurrWeatherCard.jsx

```diff
@@ -0,0 +1,49 @@
+import styles from './CurrWeatherCard.module.scss';
+import endpoints from '../../Helpers/endpoints';
+
+export default function CurrWeatherCard(props) {
+    let locationData = JSON.parse(props.locationData);
+    let currWeather = JSON.parse(props.currWeather);
```

Comment:
would be better to pass already transformed data through props

Reply:
done